include/alpaka/core/Positioning.hpp

```diff
@@ -24,6 +24,17 @@
 namespace alpaka
 {
     //-----------------------------------------------------------------------------
+    //! Defines the parallelism hierarchy levels of Alpaka
+    //-----------------------------------------------------------------------------
+    namespace hierarchy
+    {
+        struct Grids{};
```

Comment:
Yes I will add some comments.
The braces were needed because we need to instantiate the structs to give it to the the atomic functions.